src/perf/lib/ThroughputClient.cpp

```diff
@@ -288,6 +299,31 @@ ThroughputClient::StartQuic()
             &LocalIpAddr);
     }
 
+    if (CibirIdLength) {
+        BOOLEAN Opt = TRUE;
+        Status =
+            MsQuic->SetParam(
```

Comment:
Isn't the whole point of this change to not require sharing the UDP binding? I thought that with the parameter set, all short headers would get a cid, which we're trying to avoid iirc.